### PR TITLE
compose: fix very large smoke test

### PR DESCRIPTION
**Pull request by dB2510**
Enables very large smoke test with 7 of 10 cluster and 100 validators.

category: test
ticket: #1471

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -688,7 +688,7 @@ func newETH2Client(ctx context.Context, conf Config, life *lifecycle.Manager,
 	if conf.SimnetBMock { // Configure the beacon mock.
 		const dutyFactor = 100 // Duty factor spreads duties deterministically in an epoch.
 		opts := []beaconmock.Option{
-			beaconmock.WithSlotDuration(time.Second),
+			beaconmock.WithSlotDuration(time.Second * 6),
```
Comment (corverroos) [inline app/app.go:691]: why this?

File: testutil/compose/smoke/smoke_test.go [hunk 2]
```diff
@@ -84,15 +84,16 @@ func TestSmoke(t *testing.T) {
...
 				conf.NumNodes = 10
 				conf.Threshold = 7
-				conf.NumValidators = 15
+				conf.NumValidators = 100
 				conf.InsecureKeys = true
 				conf.KeyGen = compose.KeyGenCreate
+				conf.VCs = []compose.VCType{compose.VCMock}
 			},
-			Timeout: time.Second * 120,
+			Timeout: time.Minute * 5,
```
Comment (corverroos) [inline testutil/compose/smoke/smoke_test.go:96]: 5min is very long

File: app/app.go (hunk 1 above)
Comment (dB2510) [reply app/app.go:691]: I tested with slot times 3s and 6s. And found out this large cluster is stable with 6s slot times. Whereas in 3s slot times, I observed 70% of duties were failing and those that were passing had broadcast delay of around 9s.
With 6s slot times the cluster had broadcast delays of around 5.5s to 6.5s. This is only for simnet/compose run and not for simnet_test.

Comment (corverroos) [reply app/app.go:691]: I would prefer 1s for compose by default, we can do 6s only for this test?

Comment (dB2510) [reply app/app.go]: Done!

File: testutil/compose/smoke/smoke_test.go (hunk 2 above)
Comment (dB2510) [reply testutil/compose/smoke/smoke_test.go]: updated to 2 mins

**Review by corverroos — APPROVED** [review]

**Pull request merged**